indexer/ingests_indexer/src/main/resources/application.conf

```diff
@@ -0,0 +1,8 @@
+aws.metrics.namespace=${?metrics_namespace}
+aws.sqs.queue.url=${?queue_url}
+es.host=${?es_host}
+es.port=${?es_port}
+es.username=${?es_username}
+es.password=${?es_password}
+es.protocol=${?es_protocol}
+es.ingests.indexPrefix=${?es_ingests_index_prefix}
```

Comment:
this is the most spectacularly miniscule nit which you are free to ignore, but as a point of trivia Lighbend recommend using `kebab-case` for HOCON variable names https://github.com/lightbend/config/blob/master/HOCON.md#hyphen-separated-vs-camelcase